Plugin/src/main/resources/plugin.yml

```diff
@@ -38,6 +38,7 @@ permissions:
       rosestacker.stacktool.give: true
       rosestacker.silktouch: true
       rosestacker.spawnerconvert: true
+      rosestacker.break: true
```

Comment:
This permission name seems really weird. One already exists named rosestacker.silktouch, so why not rosestacker.silktouch.peotect or something?

Reply:
alright will do that then 👍

Reply:
also cant put it under rosestacker.silktouch because it has nothing to do with silktouching
it is purely for if you are able to break spawners or not
do you have any ideas for a permission node for that?

Reply:
perm name change to rosestacker.spawnermine